### PR TITLE
BZ1960541: annotations

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.6+ versions
- [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1960541)
- [Preview](https://deploy-preview-39612--osdocs.netlify.app/openshift-enterprise/latest/networking/routes/route-configuration)
- @arvind iyengar

### CODE REVIEW

File: modules/nw-route-specific-annotations.adoc [hunk 1]
```diff
@@ -35,6 +35,8 @@ Note: Using this annotation provides basic protection against distributed denial
 |`haproxy.router.openshift.io/rate-limit-connections.rate-tcp`| Limits the rate at which a client with the same source IP address can make TCP connections. It accepts a numeric value.  +
 Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks. |
 |`haproxy.router.openshift.io/timeout` | Sets a server-side timeout for the route. (TimeUnits) | `ROUTER_DEFAULT_SERVER_TIMEOUT`
+|`haproxy.router.openshift.io/timeout-tunnel` | This timeout is effective in the case of tunnel connection, for example, WebSocket over cleartext, edge, reencrypt, or passthrough routes. With cleartext, edge, or reencrypt route types, this annotation is applied as a timeout tunnel with the existing timeout value. For the passthrough route types, the annotation takes precedence over any existing timeout value set. | `ROUTER_DEFAULT_TUNNEL_TIMEOUT`
```
Comment (mjpytlak) [inline modules/nw-route-specific-annotations.adoc:38]:
```suggestion
|`haproxy.router.openshift.io/timeout-tunnel` | This timeout applies to a tunnel connection, for example, WebSocket over cleartext, edge, reencrypt, or passthrough routes. With cleartext, edge, or reencrypt route types, this annotation is applied as a timeout tunnel with the existing timeout value. For the passthrough route types, the annotation takes precedence over any existing timeout value set. | `ROUTER_DEFAULT_TUNNEL_TIMEOUT`
```
I am not sure if my suggest changes the technical accuracy of the sentence, but I did stumble over `effective`.

File: modules/nw-route-specific-annotations.adoc [hunk 2]
```diff
@@ -35,6 +35,8 @@ Note: Using this annotation provides basic protection against distributed denial
...
+|`ingresses.config/cluster ingress.operator.openshift.io/hard-stop-after` | You can set either an ingresscontroller or the ingress config . Redeploys the router and configures the HA proxy to emit the haproxy `hard-stop-after` global option, which defines the maximum time allowed to perform a clean soft-stop. | `ROUTER_HARD_STOP_AFTER`
```
Comment (mjpytlak) [inline modules/nw-route-specific-annotations.adoc:39]:
```suggestion
|`ingresses.config/cluster ingress.operator.openshift.io/hard-stop-after` | You can set either an ingresscontroller or the ingress config . This annotation redeploys the router and configures the HA proxy to emit the haproxy `hard-stop-after` global option, which defines the maximum time allowed to perform a clean soft-stop. | `ROUTER_HARD_STOP_AFTER`
```

**Review by mjpytlak — CHANGES_REQUESTED** [review]

File: modules/nw-route-specific-annotations.adoc (hunk 2 above)
Comment (sagidlow) [reply modules/nw-route-specific-annotations.adoc:39]: One other comment, should ingresscontroller be in backticks? For example `ingresscontroller`? Just something to think about and review. If not, then you can ignore this comment :)

**Comment by quarterpin** [comment]
The changes suggested by the team need to be added (as of writing). Overall, with the recommendations in place, it looks good to me. Thanks.

File: modules/nw-route-specific-annotations.adoc (hunk 2 above)
Comment (Amrita42) [reply modules/nw-route-specific-annotations.adoc:39]: Based on context I see instances of backticks being used for example IngressController resource or object or API. Based on the context I would not add backticks but I realized the spelling is wrong >> IngressController (correct one) . :)

**Comment by mjpytlak** [comment]
Peer review LGTM.

**Pull request merged**